src/index.html

```diff
@@ -5,8 +5,390 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Duplicate link tags for Google fonts are present, which are not necessary and can slow down page load times.

Reply:
not fixed